install/deploy_final_non-compute_node.md

```diff
@@ -154,43 +154,22 @@ The steps in this section load hand-off data before a later procedure reboots th
             /var/log/conman \
             /var/log/zypper.log 2>/dev/null |
          sed 's_^/__' |
-         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-$(date +%Y-%m-%d_%H-%M-%S).tgz"
+         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-$PITBackupDateTime.tgz"
     ```
 
-1. (`pit#`) Backup the bootstrap information from `ncn-m001`.
+1. (`pit#`) Backup the bootstrap information from `ncn-m001` before reboot.
 
-    > **`NOTE`** This preserves information that should always be kept together in order to fresh-install the system again.
+    > **`NOTE`** This preserves information that should always be kept together. This data can be critical for debug and support as well as in order to fresh-install the system again.
 
-    1. Log in and set up passwordless SSH **to** the PIT node.
+   ```bash
+   PITBackupDateTime=$(date +%Y-%m-%d_%H-%M-%S); cray artifacts create config-data pit-logs-backup-$PITBackupDateTime.tgz ${PITDATA}/prep/logs/pit-backup-$PITBackupDateTime.tgz
```

Comment:
It doesn't seem like you'd want to set this variable again here, or it will not match the value from earlier, and therefore the file will not exist.

Reply:
Correct, there were a few changes required I found Friday iI didn't get to update. This was one of them. Correcting that.

Reply:
Okay -- if you're still working on it and don't want it reviewed yet, you can keep it in draft mode. I think most of us only look at the ones that are marked as ready for review. Just to avoid you getting review comments on stuff you're already aware of